src/main/java/baron/Utils/DateUtils.java

```diff
@@ -0,0 +1,38 @@
+package baron.Utils;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Manages parsing and formatting of dates
+ */
+public class DateUtils {
+
+    // 03/21/1999 1600
```

Comment:
I like how you have included an example DateTime format to allow other developers to have an understanding of the format required!